Docs/README.md

```diff
@@ -39,61 +40,17 @@ This application takes an image of the leaf or crop as input and shows the resul
 
 
 
-
 ## Important Notice
 
 **Crop Ai** has been selected at [GirlScript Summer of Code 2020](https://www.gssoc.tech/)!!!
 
 >Only GSSoC participants are allowed to contribute to this repository.
 
 <p align="center">
-  <img width="300" height="300" src="./designs/logo-designs/gssoc-logo.png">
+  <img width="300" height="300" src="../designs/logo-designs/gssoc-logo.png">
 </p>
 
-
-## :four_leaf_clover: How to start contributing 
-
-
-**1.** Fork [this](https://github.com/CropAi/Frontend) repository.
-
-**2.** Clone the forked repository.
-
-```terminal
-git clone https://github.com/<your-github-username>/CropAi/Frontend.git
-```
-
-**3.** Navigate to the project directory.
-
-```terminal
-cd Frontend
-```
-
-**4.** Create a new branch.
-
-```terminal
-git checkout -b <your_branch_name>
-```
-
-**5.** Make changes in source code.
-
-**6.** Commit your changes.
-
-```terminal
-  git add .
-  git commit -m "<your_commit_message>"
-```
-
-**7.** Push your local branch to the remote repository.
-
-```terminal
-git push -u origin <your_branch_name>
-```
-
-**8.** Create a Pull Request!
-
-**Congratulations!**  :boom: You have made your first contribution
- 
-
+Before contributing to this project do check CONTRIBUTING.md file and CODE_OF_CONDUCT.md file.
```

Comment:
Add links to CONTRIBUTING.md and CODE_OF_CONDUCT.md , on clicking file name the respective file should open.